aur.py

```diff
@@ -15,6 +15,18 @@ def package_installed(module, package_name):
     return exit_code == 0
 
 
+def update_packages(module, tool):
+    assert tool in TOOL_TO_INSTALL_CMD_MAP
+
+    cmd = ['env', 'LC_ALL=C'] + TOOL_TO_INSTALL_CMD_MAP[tool] + ['--update']
```

Comment:
```
% yaourt --update
pacman: unrecognized option '--update'
```

Reply:
yes indeed tested with pacaur but for yaourt i need to change the syntax